cypress/integration/rendering/pie.spec.js

```diff
@@ -75,4 +75,20 @@ describe('Pie Chart', () => {
       expect(svg).to.not.have.attr('style');
     });
   });
+
+  it('should render a pie diagram with given outside stroke width', () => {
+    renderGraph(
+      `
+    pie title Sports in Sweden
+       "Bandy" : 40
+       "Ice-Hockey" : 80
+       "Football" : 90
+      `,
+      { pie: { outerBorderWidth: 5 } }
+    );
+    cy.get('.pieOuterCircle').should((circle) => {
+      const strokeWidth = parseFloat(circle.attr('stroke-width'));
+      expect(strokeWidth).to.eq(5);
+    });
+  });
```

Comment:
I'd like to add one but, the only value changed by the text-position parameter are the label coordinates.

Any suggestions on what a test for this might look like, that doesn't duplicate the rendering code?

The X/Y positions is given by the arc generator, based on the user parameter and the radius
The radius is based on the width/height/margin

Reply:
You can add an `imgSnapshotTest`, which verifies the visual output.

Reply:
Added

Reply:
Cool!